src/main/java/frc/robot/subsystems/shooterSubsystem/ShooterSubsystem.java

```diff
@@ -303,7 +306,7 @@ public double getSpeedRightInRPM() {
           Constants.ShooterConstants.SHOOTER_IDLE_SPEED_RIGHT);
 
   public Trigger getBeamBreak() {
-    return new Trigger(beamBreak::get);
+    return new Trigger(() -> false);
```

Comment:
Is the beam break working so we can revert this?
Auton needs this to shoot preload